tools/sidekick/config.json

```diff
@@ -2,6 +2,7 @@
   "project": "Creative Cloud",
   "previewHost": "main--cc--adobecom.aem.page",
   "liveHost": "main--cc--adobecom.aem.live",
+  "trustedHosts": ["milo.adobe.com"],
```

Comment:
@andreituicu - shouldn't we add `https://main--milo--adobecom.aem.page` here to support https://github.com/adobecom/cc/blob/b77c5c82f8fbd39d7f51717f2602c249e529655b/tools/sidekick/config.json#L37C15-L37C52? The URL should have been `milo.adobe.com`, but I don't want to break things

Reply:
Fully agree! Done in [e60f5ed](https://github.com/adobecom/cc/pull/718/commits/e60f5edf69372c13339b1ac724acedfe5d25d6f3).